docs/create_dimensions.md

```diff
@@ -13,12 +13,35 @@ In order to successfully push your JSON schema into the Monitor platform, you wi
 1. In the hamburger menu on the left side of the screen of Maximo Asset Monitor, click `Connect`.
 2. To the right of where it says `Watson IoT Platform Service` click `View Details`.
 3. Now click the `Launch` button.
-4. 
-
-## Creating a Value Card
-
-## Uploading an Image
-
-## Creating an Image Card
-
-## Creating a Table Card
+4. Sign into the `vrvzh6` organization in the Watson IoT Platform Service window. You can locate the organization on the top right corner of your screen. Click in the red boxed area on your browser to change the organization if need be. ![Changing IoTP Org](img/cd2.png) 
```

Comment:
People will not be able to do the lab if they don't have access to VRZH6. I recommend you suggest the alternative if they have their own IOT ORg.